Framework/Algorithms/src/Rebin2D.cpp

```diff
@@ -126,22 +128,23 @@ void Rebin2D::exec() {
   {
     PARALLEL_START_INTERUPT_REGION
 
+    m_progress->report("Computing polygon intersections");
```

Comment:
You moved this out of the inner loop (for performance?), but if you do so shouldn't you also adapt the `nreports` variable that gives the length of the progress bar?